main-app/app/scripts/gameboard/gameboard-services/gameboard-service.js

```diff
@@ -0,0 +1,41 @@
+angular.module('Server.MyModule')
+    .service('FreshGame', ['PlayerService','NewGame','CoreData','ServerTurn','$state',  function (playerService, newGame, coreData, serverTurn, $state) {
+        var outcome = 'Continue',
+            me = this,
+            gameEnded = function () {
+                if (outcome === "Win") {
+                    $state.go('route4');
+                }
+                else if (outcome === "Draw") {
+                    $state.go('route5');
+                }
+            },
+            updateInformaton = function (response) {
+                coreData.gameBoard = response.gameboard;
+                coreData.currentGameState = response.outcome;
+                outcome = response.outcome;
+                coreData.winner = response.winner;
+            };
+
+        me.startGame = function () {
+            me.Player1 = playerService.player1;
```

Comment:
don't "copy" the values like this - just use in the playerService. You've got one canonical source - stick with it,